lib/services/access_request.go

```diff
@@ -1176,12 +1175,8 @@ func (m *RequestValidator) Validate(ctx context.Context, req types.AccessRequest
 
 		now := m.clock.Now().UTC()
 
-		// Calculate the expiration time of the Access Request (how long it
-		// will await approval).
-		ttl, err := m.requestTTL(ctx, identity, req)
-		if err != nil {
-			return trace.Wrap(err)
-		}
+		// Max TTL for an access request is one week.
+		ttl := 7 * 24 * time.Hour
```

Comment:
Is this a "max TTL" or is it now the only TTL?

Reply:
You're right, it's the only TTL.